### ISSUE TITLE
Anyone else got things reset today?

### ISSUE THREAD

**Issue by doronazl** [itchannel/fordpass-ha#485]
Woke up to nothing working today, had to reset everything including from the car side.
Even my ford user was gone, i had to register from scratch.
Is ford messing about again?

**Comment by doronazl** [comment]
Moreover my door lock status is all wrong now.
Always showing unlocked even when it's locked

**Comment by ziptbm** [comment]
> Woke up to nothing working today, had to reset everything including from the car side. Even my ford user was gone, i had to register from scratch. Is ford messing about again?

This happened to me a few weeks ago, but luckily it was with my second account that I use with HA.  I received emails after midnight one night letting me know that this other "driver" had been removed from each of my vehicles and the login no longer worked.  I had to re-register the account from scratch and then reconfigure each of my vehicles with that ID.  At the time, I had it setup with my same first and last name, so I'm hoping that it now won't do that since it's set up as another driver.

Not sure if related, but I'm glad it was my HA account and not my main one.
